spec/datadog/core/telemetry/v1/application_spec.rb

```diff
@@ -107,4 +107,43 @@
       end
     end
   end
+
+  describe '#to_h' do
+    subject(:to_h) { application.to_h }
+
+    let(:env) { 'prod' }
+    let(:language_name) { 'ruby' }
+    let(:language_version) { '3.0' }
+    let(:products) do
+      Datadog::Core::Telemetry::V1::Product.new(
+        appsec: Datadog::Core::Telemetry::V1::AppSec.new(version: '1.0'),
+        profiler: Datadog::Core::Telemetry::V1::Profiler.new(version: '1.0')
+      )
+    end
+    let(:runtime_name) { 'ruby30' }
+    let(:runtime_patches) { 'patch' }
+    let(:runtime_version) { '3.2.1' }
+    let(:service_name) { 'myapp' }
+    let(:service_version) { '1.2.3' }
+    let(:tracer_version) { '1.0' }
+
+    before do
+      allow(products).to receive(:to_h).and_return({ profiling: { version: '1.0' }, appsec: { version: '1.0' } })
+    end
+
+    it do
+      is_expected.to eq(
+        { env: env,
+          language_name: language_name,
+          language_version: language_version,
+          products: { profiling: { version: '1.0' }, appsec: { version: '1.0' } },
+          runtime_name: runtime_name,
+          runtime_patches: runtime_patches,
+          runtime_version: runtime_version,
+          service_name: service_name,
+          service_version: service_version,
+          tracer_version: tracer_version }
+      )
+    end
```

Comment:
Minor: You can actually omit the `{` / `}`here (like you did for other tests, actually).

(Note: This also applies to other files in this PR)